Rust/src/nodestore.rs

```diff
@@ -21,7 +24,15 @@ use crate::{
 */
 
 #[repr(C)]
-pub struct NewNodeStore<C, P, N> {
+pub struct VectorNodeStore<C, P, N>
```

Comment:
VectorizedNodesStore? One reason for not using structs has been that we use different sizes for the left(right), cut and mass (structs may force a loss because if packing/alignment).

Reply:
Yes, I changed it for `VectorizedPointStore` but forgot this one. Will fix in the next PR.